### ISSUE TITLE
Official Cosmos ATOM-ONE NFT

### ISSUE THREAD

**Issue by COverS8** [atomone-hub/genesis#20]
Distributed at the genesis for everyone as an OG bluemark which can have later usecases and hold future benefits

**Comment by giunatale** [comment]
I appreciate the spirit, but let's not turn this into a degenfest. A POAP-like NFT maybe, but then why? If you get into the genesis, there's your OG mark.
Giving usecases and future benefits? I really don't like the idea. Let's be frank, these are usually marketing gimmicks. And we don't want those IMHO.

**Comment by jaekwon** [comment]
Everyone even YES voters would get some $ATOM1. 
It could be fun and interesting to create an NFT for NO+NWV voters for 82 and/or 848 (and not YES for either).
See related: https://github.com/atomone-hub/genesis/issues/21
It could be useful to have an "OG AtomOne" badge. There were only less than 8000 NO+NWV voters for prop 848.

But also this could be done permissionlessly, so maybe doesn't belong at least not in the founding documents. Maybe what we need is another repo to host creative fun ideas independent of the genesis repo.

**Comment by COverS8** [comment]
Yeah it would be great memory too and we have these NFT enthusiastics all over the Cosmos who would gladly design one

**Comment by jaekwon** [comment]
Closing because it is out of scope for genesis. We can explore it later.
